.github/workflows/linux.yml

```diff
@@ -1046,6 +1099,7 @@ jobs:
           TEST_PRECISION: FP16
```

Comment:
looks like conditions for tests above can also be added

Reply:
Done (I hope I've covered more or less everything we have)